lib/features/support/servlet.rb

```diff
@@ -67,6 +68,35 @@ def log_request(request)
       $logger.debug "BODY: #{JSON.pretty_generate(JSON.parse(request.body))}"
     end
   end
+
+  def check_digest(request)
+    header = request['Bugsnag-Integrity']
+    return if header.nil?
+
+    # Header must have type and digest
+    parts = header.split ' '
+    raise "Invalid Bugsnag-Integrity header: #{header}" unless parts.size == 2
+
+    # Both digest types are stored whatever
+    sha1 = Digest::SHA1.hexdigest(request.body)
+    simple = request.body.length
```

Comment:
This also needs to be `bytesize`

```suggestion
    simple = request.body.bytesize
```